src/plugins/workspace/public/components/workspace_creator/workspace_form.tsx

```diff
@@ -109,7 +117,9 @@ export const WorkspaceForm = ({
   const [icon, setIcon] = useState(defaultValues?.icon);
   const [defaultVISTheme, setDefaultVISTheme] = useState(defaultValues?.defaultVISTheme);
 
-  const [selectedFeatureIds, setSelectedFeatureIds] = useState(defaultValues?.features || []);
+  const [selectedFeatureIds, setSelectedFeatureIds] = useState(
+    defaultValues?.features || DEFAULT_CHECKED_FEATURES_IDS
```

Comment:
@wanglam @SuZhou-Joe According to your two comments on this section, it seems that we should make `defaultValues?.features` append defaultCheckedFeatures, but there will be two problems:
1. The logic of append will only be triggered when updating, even if the user does not submit the form, it will not take effect.
2. It seems that only the dirty data in the development stage needs such compatibility logic. Is it necessary for us add logic to complicate development dirty data?
What's your option?

Reply:
Since the API won't deny create or update workspace without these default feature ids and won't append these automaticlly. I think we need to update the left navigation code to always show these features in it.

Reply:
I think if users use API to create workspace manually, we can't stop it because it's not a regular way. But i can add complicate logic to append the default checked features into selectedFeatureIds.